mock/process.go

```diff
@@ -19,8 +19,8 @@ type Process struct {
 	WaitExitCode                int
 
 	ProcInfo         jasper.ProcessInfo
-	Triggers         jasper.ProcessTriggerSequence
-	SignalTriggers   jasper.SignalTriggerSequence
+	Triggers         jasper.ProcessTriggerSequence `json:"-"`
```

Comment:
I don't have an objection to this, but why do these two fields need JSON omission tags (and not the others)?

Reply:
Because ProcessTrigger is a function that can't be serialized. I think the actual behavior is that this just silently gets omitted but the static checker flags it to specifically omit. I can add a comment since this is maybe confusing.

Reply:
Added a comment.

Reply:
Let me look into why this is being json serialized... although it might be overzealous static checking where it _might_ be serialized as opposed to actually being...

Reply:
It was being json serialized to be used as "invalid" input in cli/manager_test.go. I switched to using an arbitrary invalid map instead, so we don't need the weird json tagging.